File: jsunit/ZombieDicePoolTest.js
ZombieDicePoolTest = TestCase("zombieDicePoolTest");

ZombieDicePoolTest.prototype.testPoolsShouldBeSeperate = function() {
	var pool1 = new ZombieDicePool();
	var pool2 = new ZombieDicePool();
	
	assertEquals(13, pool1.getCount());
	assertEquals(13, pool2.getCount());
};

File: junit/se/fredsberg/game/zombiedice/ZombieDicePoolTest.java
package se.fredsberg.game.zombiedice;

import static org.junit.Assert.*;

import org.junit.Test;

public class ZombieDicePoolTest {

    @Test
    public void count_shouldBe13() {
        assertEquals(13, new ZombieDicePool().getCount());
    }
}
